### PR TITLE
Scenario - 1 Completed 

### PR DESCRIPTION
**Pull request by chiragkhimani**
Please review the code and feel free to provide suggestions

### CODE REVIEW

**Comment by chiragkhimani** [comment]
I reviewed it and looks good +1

**Review by chiragkhimani — COMMENTED** [review]
Looks good

**Comment by guljahanJ** [comment]
looks great!

**Comment by Fidema** [comment]
> 
> 
> Please review the code and feel free to provide suggestions

I reviewed the code and looks good + Fidan

**Comment by vildanembel** [comment]
fantastic !

**Comment by AishaGench** [comment]
that's great

**Comment by ElifQA** [comment]
perfect!

File: src/test/resources/features/AddEmployee.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Feature: This feature we created to test sign up functionality
+
+  Scenario: Verify admin can create username
+    Given user open orange hrm website
+    When user login with admin access
+    Then verify user is on home page
```
Comment (chiragkhimani) [inline src/test/resources/features/AddEmployee.feature:6]: "verify user is on home page" this step is not clear please write in details.

Comment (chiragkhimani) [reply src/test/resources/features/AddEmployee.feature:6]: done!

**Comment by Fidema** [comment]
@chiragkhimani @guljahanJ @vildanembel 

 good job!

**Pull request merged**